apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -40,8 +36,20 @@ def add_bool(self, node):
 
 def _load_yaml(path: Path):
     """Helper function for reading a YAML file"""
-    with open(path, encoding="utf-8") as yaml_file:
-        return load(yaml_file, Loader=SafeLoader)
+    with open(path) as yaml_file:
+        try:
+            from yaml import CSafeLoader
+
+            return load(yaml_file, Loader=CSafeLoader)
+        except ImportError:
```

Comment:
How come this logic works here, but it didn't work when it was on the top of the file? :thinking:

Reply:
TBH I ended up with this not so elegant approach to avoid mypy errors.

If it is OK to add explicit `# type: ignore` on the import statements, I can go back to the original top-of-the-file approach.

Reply:
Well, if this has working types then it's better